helix/utils.py

```diff
@@ -655,6 +655,44 @@ def source(package, resource):
     return content
 
 
+def extract_fnames(src):
```

Comment:
Is this only used by the Tigress transform? If so, I think it belongs in that module - we can promote it to the `utils` module if/when it's used in multiple places